Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -2,50 +2,63 @@
 
 import com.test.api.dto.CategoryDTO;
 import com.test.api.execution.BaseTest;
-import org.junit.jupiter.api.Test;
-
-import java.util.Arrays;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.Tag;
+import java.util.*;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
+import static org.assertj.core.api.Assertions.assertThat;
 
-/**
- * Test for Category API
- */
 public class CategoryTest extends BaseTest {
 
-    /**
-     * Test ob category title verification
-     */
+    private int idCategoryToClean = 0;
+    private final String idQueryParam = "id";
+
+    @AfterEach
+    void cleanup(TestInfo testInfo) {
+        if (testInfo.getTags().contains("needs-cleanup")) {
+                given().spec(defaultRequestSpec())
+                        .queryParam(idQueryParam , idCategoryToClean)
+                        .when()
+                        .delete(getTestEnvironment().getCategoryPath());
+        }
+    }
+
+
     @Test
-    public void verifyGetCategory() {
+    public void canGetCategoryByCorrectId() {
         final int categoryId = 5;
         final String expectedTitle = "Street";
 
-        assertThat("Category title is unexpected",
+        assertThat("Category title should be %s, but is %s",
```

Comment:
this expression should use String.format, otherwise when test fails, the message is: 
java.lang.AssertionError: Category title should be %s, but is %s